pkgs/top-level/python-packages.nix

```diff
@@ -3191,6 +3193,10 @@ in {
 
   neuronpy = callPackage ../development/python-modules/neuronpy { };
 
+  panflute = callPackage ../tools/text/panflute { };
```

Comment:
please read the header of this file

Reply:
@FRidh: apologies, moved it to all-packages. As a separate commit I swapped an out-of-order entry, don't know if that is desirable otherwise I can revert.